source/renderer/zettlr-editor.js

```diff
@@ -126,6 +127,11 @@ class ZettlrEditor {
 
     this._spaceWidth = 0
     this._monospaceWidth = 0
+    ipc.on('toggle-theme', (e) => this.resetSpaceCalculation())
+    ipc.on('switch-theme-berlin', (e) => this.resetSpaceCalculation())
+    ipc.on('switch-theme-bielefeld', (e) => this.resetSpaceCalculation())
+    ipc.on('switch-theme-frankfurt', (e) => this.resetSpaceCalculation())
+    ipc.on('switch-theme-karl-marx-stadt', (e) => this.resetSpaceCalculation())
```

Comment:
Better approach to this: Only listen to the config-change event (I don't know exactly what it's called, but the `config-provider.js` emits an event as soon as the config has changed — then check if the value of the theme-config-value is different and handle this instead. Keeps the boilerplate a little bit in check! :)

Reply:
Could we make another event that's fired on theme switch and on customcss update? Smth like `update-style`.

Reply:
Puhhh… until I manage to refactor all the boilerplate in the IPC classes, I'm hesitant in adding new events.

After all, I'm currently planning to remove these `message` IPC calls altogether and replace them with appropriate channels (e.g. `config` for config-events, `fsal` or similar for file system changes, etc …). This would make it more manageable, modular and decentral. But I have to refactor that damned file system beforehand :D

Reply:
Ok, done!